02_configure_host.sh

```diff
@@ -61,23 +61,23 @@ else
       fi
       sudo ifdown provisioning || true
       sudo ifup provisioning
-  
+
```

Comment:
Nit: ideally unrelated formatting changes would be split out into another commit. You can leave it though

Reply:
ok, sorry